cmake/Buildwearables.cmake

```diff
@@ -6,6 +6,7 @@ include(FindOrBuildPackage)
 
 find_or_build_package(YARP QUIET)
 find_or_build_package(iDynTree QUIET)
+find_or_build_package(YARP_telemetry QUIET)
```

Comment:
You need to also add `YARP_telemetry` to the `DEPENDS` argument of `ycm_ep_helper` (probably we should eventually  modify `ycm_ep_helper` to avoid having two places where to list the dependency.

Reply:
I tried compiling without that in my local machine, looks like the `wearables` is compiling fine with IWearLogger enabled.

Reply:
Maybe because it contained as one of the components in YARP?

Reply:
The DEPENDS just ensure that `wearables` is build **after** `YARP_telemetry`. It is perfectly possible that in your case, by chance  `YARP_telemetry` is built and installed before `wearables` and so everything works fine, but we should add `YARP_telemetry` to ensure that this happens.

Reply:
Ah understood.

Reply:
I rebased with the suggested change in https://github.com/robotology/robotology-superbuild/pull/730/commits/a6931f3e7da85cb1b5fb997979b2bfce59c7af1f